controller/src/main/java/org/jboss/as/controller/logging/ControllerLogger.java

```diff
@@ -3349,4 +3349,7 @@ XMLStreamException discoveryOptionsMustBeDeclared(String adminOnlyCmd, String po
 
     @Message(id = 404, value = "The binding name '%s' in socket binding group '%s' is not unique. Names must be unique across socket-binding, local-destination-outbound-socket-binding and remote-destination-outbound-socket-binding")
     OperationFailedException bindingNameNotUnique(String name, String groupName);
+
+    @Message(id = 405, value = "Couldn't build the report")
+    OperationFailedException failedToBuildReport(@Cause Throwable t);
```

Comment:
This should return RuntimeException. An OFE is a form of OperationClientException and shouldn't be thrown except in cases where the client has made a mistake in their request. An OSH should throw a RuntimeException if it encounters an unexpected server fault not due to client error.

AbstractOperationContext.executeStep counts on this in its error handling; e.g. after boot it will only log an OFE in the server log at DEBUG.